config/environments/production.rb

```diff
@@ -87,4 +87,8 @@
   #
   # Skip DNS rebinding protection for the default health check endpoint.
   # config.host_authorization = { exclude: ->(request) { request.path == "/up" } }
+
+  config.assets.compile = false
+config.assets.digest = true
```

Comment:
We do not need to change `production.rb` env file now. We work only in dev env.

Please remove node_modules from git and add to .gitignore
```
git rm -rf node_modules

echo /node_modules >> .gitignore

```